nb2/api.py

```diff
@@ -52,3 +52,35 @@ def create_person():
     response.status_code = 201
 
     return response
+
+@bp.route('/quotes', methods=['POST'])
+def create_quote():
+    data = request.get_json() or {}
+    slack_user_id = data.get('slack_user_id')
+
+    target_person = Person.query.filter(Person.slack_user_id==slack_user_id).one_or_none()
+
+    if not target_person:
+        error_msg = f"Can't add a quote to Person with slack_user_id {slack_user_id} " \
+                     "because they don't exist."
+        return validation_error(error_msg)
+
+    required_field_errors = Validators.validate_required_fields_are_provided(Quote, data)
```

Comment:
Just one more small comment from something that came up while devQAing. It might be nicer to do this check before checking if the person exists. If I POST an empty body, I get the error that the person None doesn't exist, rather than just an error telling me fields are missing.

Reply:
Right, good point - changed